eeg_cofitage/README

```diff
@@ -0,0 +1,71 @@
+﻿Overview
+--------
+
+This is the “COFITAGE” dataset. The aim is to collect sleep EEG data from 100 to 120 healthy older 
+participants (aged 50-70), in order to quantify sleep disruption, evaluate its impact on cognition
+and assess its relationship with brain amyloid and tau proteins deposition in preclinical AD.
+
+We would like to determine which sleep-wake regulation parameters are associated with biomarkers 
+and cognitive status at baseline and at a 24 months follow-up of cognitive performances.
+
+
+Citing this dataset
+-------------------
+
+This dataset and related analyses are unpublished. Please do not use/cite given data other than
+BIDS dataset example.
+
+
+Format
+------
+
+It is formatted according to BIDS version 1.2.0 and the BIDS standard extension
+for EEG (BEP006). The full description of the standard and extension which can be found here:
+https://bids-specification.readthedocs.io/en/latest/
+
+This dataset is originally created in Embla 3.4 format. It is converted to EDF+ and applied to BIDS standard by a private tool. The configuration and log files are stored in the sourcedata folders.
+
+
+Details about the experiment
+----------------------------
+
+In the COFITAGE project, EEG data was collected from volunteers aged 50-70.
```

Comment:
Please mention how many subjects are part of this dataset: What is `N`?

Reply:
Will do

Reply:
Just for clarification: Are you planning to wait with merging this dataset until it's "completed"? Or do you want this example to be merged and then you want to update it bit by bit?

Reply:
Not sure that the number of subject will mean anything.
The list of subjects is in `participants.tsv` for current subset, and total number for full dataset is unknown as recordings still continue.

I would like to merge to master this partial dataset, as example it doesn't really matters if there are 10 or 100 subjects.  I want to use the merging as confirmation that I produce a valid BIDS

Reply:
>  I want to use the merging as confirmation that I produce a valid BIDS

We don't have `bids-examples` for people to confirm that they are producing valid BIDS datasets. We have it to offer users a perspective on how they can organize their data.

I would accept merging this dataset, if you plan to keep it updated as the true dataset of that study keeps coming in.

Reply:
From the point of view of example, I doubt you will gain something with additional subjects -- it will make just more `sub-*` folders with same tasks.

If your policy is to put only publicly available data as example, then I afraid it will not happen this year. So I will understand if you refuse the merge.

But in this case I would like a some kind of confirmation that the created  structure is valid BIDS. The validator is great, but still in development, and nothing is worth more than confirmation a real developer of standard.

Reply:
> From the point of view of example, I doubt you will gain something with additional subjects -- it will make just more sub-* folders with same tasks.

agreed

> If your policy is to put only publicly available data as example, then I afraid it will not happen this year. So I will understand if you refuse the merge.

I think it might be better not to merge it for now

> But in this case I would like a some kind of confirmation that the created structure is valid BIDS. The validator is great, but still in development, and nothing is worth more than confirmation a real developer of standard.

We have gone through several iterations with this, and the quality has improved each time. I can't say for sure whether it's 100% perfect (I might have missed something), but the dataset is certainly BIDS compatible and in a good state

Reply:
> We have gone through several iterations with this, and the quality has improved each time. I can't say for sure whether it's 100% perfect (I might have missed something), but the dataset is certainly BIDS compatible and in a good state

Thanks a lot, now I can sleep)
The README, dataset_description, and labeling are actually user inputs, so I don't really worry about them.